src/app/core/services/navigation/nav-tree.service.ts

```diff
@@ -36,23 +38,31 @@ export abstract class NavTreeService<ContentT extends RoutedContentInfo> {
       distinctUntilChanged(),
       map(() => undefined),
     ),
-    this.content$.pipe(this.childrenNavData())
+    this.content$.pipe(this.childrenNavData()),
   );
   state$ = combineLatest([ this.children$, this.content$ ]).pipe(
     mergeScan((prevState: FetchState<NavTreeData>, [ children, content ]) => {
+      if (children instanceof Error) return of(errorState(children));
+
+      const isReload = prevState.isReady && prevState.data.selectedElementId === content?.route.id;
```

Comment:
So when fetching the menu the first time, if children response arrives (after the root), doesn't it consider this case as a reload? (while it is not)

Checking...

Before: 1 call to "activities"
![Screenshot 2022-03-16 at 10 29 18](https://user-images.githubusercontent.com/1053150/158559287-7deb20e4-0062-461d-b690-ca55eb4b5c3a.png)

After: 7 calls to "activites"
![Screenshot 2022-03-16 at 10 28 08](https://user-images.githubusercontent.com/1053150/158559056-2f2bce56-06ef-464c-8151-e28766922b68.png)

😔